clusterloader2/cmd/clusterloader.go

```diff
@@ -31,31 +30,31 @@ import (
 )
 
 var (
-	clusterConfig  config.ClusterConfig
-	testConfigPath string
+	clusterLoaderConfig config.ClusterLoaderConfig
 )
 
 func initClusterFlags() {
-	flag.StringVar(&clusterConfig.KubeConfigPath, "kubeconfig", "", "Path to the kubeconfig file")
-	flag.IntVar(&clusterConfig.NodeCount, "nodes", -1, "Number of nodes in the cluster")
+	flag.StringVar(&clusterLoaderConfig.ClusterConfig.KubeConfigPath, "kubeconfig", "", "Path to the kubeconfig file")
+	flag.IntVar(&clusterLoaderConfig.ClusterConfig.NodeCount, "nodes", -1, "Number of nodes in the cluster")
 }
 
 func validateClusterFlags() []error {
 	errList := make([]error, 0)
-	if clusterConfig.KubeConfigPath == "" {
+	if clusterLoaderConfig.ClusterConfig.KubeConfigPath == "" {
 		errList = append(errList, fmt.Errorf("no kubeconfig path specified"))
 	}
 	return errList
 }
 
 func initFlags() {
-	flag.StringVar(&testConfigPath, "testconfig", "", "Path to the test config file")
+	flag.StringVar(&clusterLoaderConfig.ReportDir, "report-dir", "", "Path to the directory where the reports should be saved. Default is empty, which doesn't generate these reports.")
```

Comment:
Shouldn't the default be to print to stdout instead of not doing anything at all?

Reply:
Description corrected.